src/app/services/files.js

```diff
@@ -113,6 +115,22 @@ module.exports = (Model, App) => {
         throw err;
       }
     }
+
+    const thumbnails = await Model.thumbnail.findAll({
+      where: { file_id: fileId }
+    });
+
+    if (thumbnails && Array.isArray(thumbnails) && thumbnails.length > 0) {
+      thumbnails.forEach(async (thumbnail) => {
```

Comment:
This forEach won't work as its intended to:

https://stackoverflow.com/questions/37576685/using-async-await-with-a-foreach-loop

Either use:
for (const thumbnail of thumbnails) {
// Save to use async here
}

or use .map and then Promise.all